src/app/common/pipes/event-type-filter.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { MetadataBlockFragment } from "src/app/api/kamu.graphql.interface";
+
+@Pipe({
+    name: "eventTypeFilter",
+})
+export class EventTypeFilterPipe implements PipeTransform {
+    transform(
+        blocks: MetadataBlockFragment[],
+        filter: string,
+    ): MetadataBlockFragment[] {
+        if (filter === "None") {
```

Comment:
This constant should be in app values. Jesus, we are making control logic dependent on magic constants :-)

Reply:
This code was deleted